markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -100,6 +100,8 @@ contract PerpsAccountModule is IPerpsAccountModule {
         FeatureFlag.ensureAccessToFeature(Flags.PERPS_SYSTEM);
```

Comment:
Although out of scope for this specific commit, I noticed `payDebt()` and `debt()` override existing functions defined within the interface `IPerpsAccountModule`.

Please use the `@inheritdoc` tag to inherit the interface documentation as is done elsewhere within the contract.

ex:
```
/**
  * @inheritdoc IPerpsAccountModule
  */
```